tracker_automations/check_marked_as_integrated/check_marked_as_integrated.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+#
+# Checks fixVersion minors against major branches commits.
+#
+#jiraclicmd: fill execution path of the jira cli
+#jiraserver: jira server url we are going to connect to
+#jirauser: user that will perform the execution
+#jirapass: password of the user
+#gitcmd: git cli path
+#gitdir: git directory with integration.git repo
+#gitremotename: integration.git remote name
+#currentmaster: the final major version current master will be (e.g. 32)
+
+# Let's go strict (exit on error)
+set -e
+
+if [ -z "$gitremotename" ]; then
+    gitremotename="origin"
+fi
+
+# Verify everything is set
+required="WORKSPACE jiraclicmd jiraserver jirauser jirapass gitcmd gitdir gitremotename currentmaster"
+for var in $required; do
+    if [ -z "${!var}" ]; then
+        echo "Error: ${var} environment variable is not defined. See the script comments."
+        exit 1
+    fi
+done
+
+# file where results will be sent
+resultfile=$WORKSPACE/check_marked_as_integrated.csv
+echo -n > "${resultfile}"
+
+# Calculate some variables
+mydir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+basereq="${jiraclicmd} --server ${jiraserver} --user ${jirauser} --password ${jirapass}"
+
+# Include some utility functions
+. "${mydir}/util.sh"
+
+# Let's search all the issues having the "ci" label that have been sent back to development (form peer review or integration)
+${basereq} --action getIssueList \
+           --search "project = 'Moodle' \
```

Comment:
ok ok, I get it now. In any case I think it is worth waiting for the script to report a failure to add the label or comment to the issue, so the integrator will still check that the reported missing branches are the expected ones, otherwise we may still miss stuff.